### PR TITLE
Update instance appName based on status:wd msgs

### PR DESCRIPTION
**Pull request by kraman**
Connect to strongloop-internal/scrum-nodeops#665

### CODE REVIEW

File: server/models/service-instance.js [hunk 1]
```diff
@@ -93,6 +93,15 @@ module.exports = function extendServiceInstance(ServiceInstance) {
   }
   ServiceInstance.recordStatusUpdate = recordStatusUpdate;
 
+  function recordStatusWdUpdate(instanceId, instInfo, callback) {
+    ServiceInstance.findById(instanceId, function(err, instance) {
+      if (err) return callback(err);
+      instance.applicationName = instInfo.appName;
```
Comment (sam-github) [inline server/models/service-instance.js]: I'd put an `if(instInfo.appName)` around this, you didn't bump a minor in supervisor, and strong-runner, and pm... which is too much work, so just not nulling applicationName if the supervisor is out of date will be a bit more robust.

**Comment by sam-github** [comment]
consider adding a conditional, otherwise LGTM

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
@slnode test please

**Pull request merged**